### PR TITLE
style : update week labelling styles and dataset settings feedback

### PR DESCRIPTION
**Pull request by keithCuniah**
# Description
Updates : 
- weak labelling styles
- messages (weak labelling and dataset settings)
- add toast on success/error saving rule

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Type of change**

(Please delete options that are not relevant. Remember to title the PR according to the type of change)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (change restructuring the codebase without changing functionality)
- [ ] Improvement (change adding some improvement to an existing functionality)
- [ ] Documentation update
- [x] styles

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes. And ideally, reference `tests`)

- [ ] Text classification
- [ ] dataset settings

**Checklist**

- [x] I have merged the original branch into my forked branch
- [ ] I added relevant documentation
- [x] follows the style guidelines of this project
- [x] I did a self-review of my code
- [ ] I made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `quickstart.Dockerfile` image default users from `team` and `argilla` to `admin` and `annotator` including new passwords and API keys ([#2564]).
 - Datasets to be managed only by users with `admin` role ([#2564]).
 - The list of rules is now accessible while metrics are computed. Closes[#2117](https://github.com/argilla-io/argilla/issues/2117)
+- Style updates for weak labelling and adding feedback toast when delete rules. Closes[#2626](https://github.com/argilla-io/argilla/pull/2626)
```
Comment (frascuchon) [inline CHANGELOG.md]:
```suggestion
- Style updates for weak labelling and adding feedback toast when delete rules. See [#2626](https://github.com/argilla-io/argilla/pull/2626)
```

**Review by frascuchon — APPROVED** [review]

File: frontend/components/text-classifier/labeling-rules/RuleEmptyQuery.vue [hunk 2]
```diff
@@ -78,8 +76,8 @@ export default {
         buttonLabels: [{ label: "Create labels", value: "CREATE_LABELS" }],
         feedbackType: "ERROR",
       },
-      messageNotLabels:
-        "To create new rules, you need al least two labels. We highly recommended starting by annotating some records with these labels.",
+      messageNotLabels: `To create new rules, you need al least two labels.
+       <br/>We highly recommended starting by annotating some records with these labels.`,
       messageNotAnnotation:
         "We highly recommended starting by annotating some records with these labels.",
```
Comment (nataliaElv) [inline frontend/components/text-classifier/labeling-rules/RuleEmptyQuery.vue:82]: Change suggestion (lines 80 & 82):
"We highly recommend starting by annotating some records with these labels."

File: frontend/components/text-classifier/labeling-rules/RuleLabelsDefinition.vue [hunk 3]
```diff
@@ -140,7 +137,7 @@ export default {
         feedbackType: "ERROR",
       },
       messageNotLabels:
-        "To create new rules, you need al least two labels. We highly recommended starting by annotating some records with these labels.",
+        "To create new rules, you need al least two labels. <br />We highly recommended starting by annotating some records with these labels.",
```
Comment (nataliaElv) [inline frontend/components/text-classifier/labeling-rules/RuleLabelsDefinition.vue:140]: Change suggestion (line 140):
"We highly recommend starting by annotating some records with these labels."

File: frontend/models/TextClassification.js [hunk 4]
```diff
@@ -424,29 +425,45 @@ class TextClassificationDataset extends ObservationDataset {
...
+          workspace: this.workspace,
+          name: this.name,
+          rules,
+          perRuleQueryMetrics,
+          rulesOveralMetrics: overalMetrics,
+        },
+      });
+    } catch (err) {
+      console.log(err);
+      message = `Rule '${query}' can't be delete`;
```
Comment (nataliaElv) [inline frontend/models/TextClassification.js:458]: Change: `Rule '${query}' can't be deleted`

**Review by nataliaElv — COMMENTED** [review]
@keithCuniah I added suggestions for a couple of copy changes.

**Pull request merged**